src/components/Article.js

```diff
@@ -72,7 +72,7 @@ const ArticleSection = ({
     }
   }
 
-  useArticleLinksNavigation('Article', lang, contentRef, linkHandlers, [page, textSize], { galleryItems, articleTitle, namespace, id })
+  useArticleLinksNavigation('Article', lang, contentRef, linkHandlers, [page], { galleryItems, articleTitle, namespace, id })
```

Comment:
`textSize` is needed here because it affects how much text is currently visible on the screen. When it changes, this hook needs to recompute which links are currently visible and can be selected.

Reply:
Yeap, currently working on this issue: when the `useArticleLinksNavigation` hook recomputes it is also triggering a `useSoftkey` update that conflicts with the `TextSize` softkeys as they both use `ArrowLeft`/`ArrowRight` keys

Reply:
I'm trying out a few ideas around the concept of adding some sort of flag or signal for when the font setting is "complete" (aka: when user presses "OKAY" CSK), but still nothing worth committing yet. This is flag we could use as the dependency for `useArticleLinksNavigation` and hence address the softkey issue. This could also help with the request of separating the functionalities of the backspace key and the CSK.

Reply:
Building from the idea above, how about https://github.com/wikimedia/wikipedia-kaios/pull/300/commits/11c9d492c044d9d2cefdb2133442cda3456e32d2? Seems to work well, although I'm trying to understand if there's a better/cleaner way.

Reply:
Came up with an idea that works better: decoupling the link navigation recomputing from the actual softkeys setting (https://github.com/wikimedia/wikipedia-kaios/pull/300/commits/4f7a3dcc59b4ca0bba68afa267fc67d4189177b0).